docs/posts/ibisml/index.qmd

```diff
@@ -0,0 +1,897 @@
+---
+title: "Using IbisML for a Kagge competition: credit risk model stability"
+author: "Jiting Xu"
+date: "2024-07-25"
+categories:
+    - blog
+    - machine learning
+    - feature engineering
+execute:
+    freeze: auto
+---
+
+## Introduction
+In this post, we'll demonstrate how to use Ibis and IbisML end-to-end for a Kaggle competition -
+[Credit risk model stability](https://www.kaggle.com/competitions/home-credit-credit-risk-model-stability).
+
+1. Load data and perform feature engineering with Ibis on a duckdb backend
```

Comment:
```suggestion
1. Load data and perform feature engineering with Ibis on a DuckDB backend
```

Reply:
Nit: You mention here that you do it on the DuckDB backend, but not for the next step; if anything, I think it's more interesting to note that you do the ML preprocessing on DuckDB (which people don't normally do, or know is possible).